sandbox

```diff
@@ -273,6 +292,7 @@ sandbox () {
     echo "* docker rmi $(docker images -f "dangling=true" -q)"
     docker rmi $(docker images -f "dangling=true" -q)          || true
     rm "$ACTIVE_CONFIG_FILE" > /dev/null 2>&1                  || true
+    echo 'clean function executed' > .clean;
```

Comment:
Same issue as above.
Simplest is to create a variable `CLEAN_FILE` that is global.